### PR TITLE
WOR-240: Create Patient IdentifierType cache

### PR DESCRIPTION
**Pull request by jomadoye**

### CODE REVIEW

File: src/features/patientIdentifierTypes/reducers.js [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+          message: "Unable to load patientIdentifierTypes"
+        }
+      };
+    default: return state;
+  }
+
+};
+
+
+export const getPatientIdentifiertypes = (state) => {
```
Comment (mogoodrich) [inline src/features/patientIdentifierTypes/reducers.js]: The "t" in "types" should be capitalized.

File: src/features/patientIdentifierTypes/reducers.js [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+  return state;
+};
+
+export const getPatientIdentifiertype = (state, patientIdentifierTypeUuid) => {
```
Comment (mogoodrich) [inline src/features/patientIdentifierTypes/reducers.js]: The "t" in "types" should be capitalized.

File: src/store.js [hunk 3]
```diff
@@ -50,6 +52,14 @@ export const selectors = {
   getConcepts: (state) => {
     return getConcepts(state.openmrs.metadata.concepts);
   },
+  
+  getPatientIdentifierType: (state, patientIdentifierUuid) => {
+    return getPatientIdentifiertype(state.openmrs.metadata.patientIdentifierTypes, patientIdentifierUuid);
+  },
+
+  getPatientIdentifierTypes: (state) => {
+    return getPatientIdentifiertypes(state.openmrs.metadata.concepts);
```
Comment (mogoodrich) [inline src/store.js]: whoops, "concepts"? :)

**Comment by jomadoye** [comment]
Yeah @mogoodrich , I'll raise a PR for the test.

**Pull request merged**